exercises/10-exercise.py

```diff
@@ -0,0 +1,103 @@
+'''
+Author: Jorge Diaz Jr
+
+Exercise 10
+-----------
+Get file size
+
+'''
+import humanize
+
+import numpy as np
+import os
+from functools import wraps
+
+# Some bluesky modules
+from bluesky import RunEngine
+from bluesky.plans import count
+
+# simulated devices from ophyd
+from ophyd.sim import (img, det1, det2, NumpySeqHandler, motor1,
+                       SynSignalWithRegistry)
+
+# databroker modules
+from databroker.tests.utils import temp_config
+from databroker import Broker
+
+
+class NumpySeqHandler:
+    specs = {'NPY_SEQ'}
+
+    def __init__(self, filename, root=''):
+        self._name = os.path.join(root, filename)
+
+
+    def __call__(self, index):
+        return np.load('{}_{}.npy'.format(self._name, index))
+
+
+    def get_file_list(self, datum_kwarg_gen):
+        #This method is optional. It is not needed for access, but for export.
+        return ['{name}_{index}.npy'.format(name=self._name, **kwargs['index']) #**kwargs
+                for kwargs in datum_kwarg_gen]
+
+
+    def get_file_size(self, datum_gen):
```

Comment:
in the spirit of the change we discussed before let's remove `get_file_size` from here and rather call `get_file_list` and figure out the file size from there (outside of the class)